### PR TITLE
Add changelog for 15.0.1

### PR DESCRIPTION
**Pull request by GeorgianaElena**

### CODE REVIEW

File: docs/source/changelog.md [hunk 1]
```diff
@@ -8,6 +8,21 @@ command line for details.
 
 ## 15.0
 
+### 15.0.1
+
+#### Bugs fixed
+
+- [Bitbucket] Fix for changes to bitbucket API - /teams removed and /workspaces to be used [#477](https://github.com/jupyterhub/oauthenticator/pull/477) ([@Marcalberga](https://github.com/Marcalberga))
+- [CILogon] Don't make action a required field of CILogonOAuthenticator.allowed_idps follow-up [#517](https://github.com/jupyterhub/oauthenticator/pull/517) ([@GeorgianaElena](https://github.com/GeorgianaElena))
+- [CILogon] Don't make action a required field of CILogonOAuthenticator.allowed_idps [#516](https://github.com/jupyterhub/oauthenticator/pull/516) ([@GeorgianaElena](https://github.com/GeorgianaElena))
```
Comment (GeorgianaElena) [inline docs/source/changelog.md:17]: Pff, what a lack of inspiration I had. @consideRatio, do you think it makes more sense to have the follow up PR be last?
like in the suggestion below.

```suggestion
- [CILogon] Don't make action a required field of CILogonOAuthenticator.allowed_idps [#516](https://github.com/jupyterhub/oauthenticator/pull/516) ([@GeorgianaElena](https://github.com/GeorgianaElena))
- [CILogon] Don't make action a required field of CILogonOAuthenticator.allowed_idps follow-up [#517](https://github.com/jupyterhub/oauthenticator/pull/517) ([@GeorgianaElena](https://github.com/GeorgianaElena))
```

Comment (consideRatio) [reply docs/source/changelog.md:17]: Oh i think you should leave it as generated by github-activity and possibly update titles/labels to ingluence how its generated. To make further tweaks seems too much work.

If you do want, feel free to make tweaks though.

**Review by consideRatio — APPROVED** [review]

**Comment by consideRatio** [comment]
An anchor is found to be broken, a link to 15.0.0 is now to be 15.0.1

**Comment by GeorgianaElena** [comment]
> An anchor is found to be broken, a link to 15.0.0 is now to be 15.0.1

@consideRatio I added a new commit removing the patch version from the migration guide entirely. I thought it was less confusing than referencing a migration guide to 15.0.1 from a 15.0.0 release. What do you think?

**Comment by consideRatio** [comment]
Thanks! Closed because of clumsy fingers

**Comment by GeorgianaElena** [comment]
Thank you ✨

**Pull request merged**